3style-text/styles.css

```diff
@@ -0,0 +1,89 @@
+/* ---font styles--- */
+
+body {
+  font-family: "Public Sans", Verdana, sans-serif;
+  font-size: 18px;
+}
+
+
+h1, h2, h3, h4, h5, h6 {
+  font-family: "Quicksand", sans-serif;
+  font-weight: 600;
+}
+
+
+/* ---font sizes--- */
+
+h1 {
+	font-size: 250%; /* 45px */
+}
+
+h2 {
+	font-size: 200%; /* 36px */
+}
+
+h3 {
+	font-size: 177.78%; /* 32px */
+}
+
+h4 {
+	font-size: 162.5%; /* 26px */
+}
+
+h5 {
+	font-size: 122%; /* 22px */
+}
+
+h6 {
+	font-size: 100%; /* 18px */
+}
+
+h3 {
+	font-weight: 700;
+}
+
+/* ---font weights--- */
+
+strong em,
+em strong {
+	font-family: "Quicksand", sans-serif;
+    font-style: normal;
+}
+
+/* ---font colors--- */
+
+body {
+	font-family: "Public Sans", Verdana, sans-serif;
+	font-size: 18px;
+	color: DarkSlateGray;
+}
+
+h1 {
+	font-size: 250%; /* 45px */
```

Comment:
Similar comment here, since you've already declared all of the font sizes, use this area to focus on color. (Alternatively, you can collapse all of these under one selector where all the styles for one thing are under one selector. But I like the thought behind this organization. This will be a helpful way to think of things as you move into variables!)